portals/publisher/source/src/app/components/Base/Header/avatar/Avatar.jsx

```diff
@@ -90,6 +91,13 @@ class Avatar extends Component {
         if (user.name.endsWith('@carbon.super') && count <= 1) {
             username = user.name.replace('@carbon.super', '');
         }
+        let usr;
+        const readOnlyUser = AuthManager.isReadOnlyUser();
+        if (readOnlyUser) {
+            usr = username + ' (read-only)';
```

Comment:
we have to make this i18n compatible, please use <FormattedMessage/> component

Reply:
![image](https://user-images.githubusercontent.com/3313885/143813485-c3de3433-dbbf-4f5b-8432-bcfc043375f5.png)
Shall we display it like this ?

[source](https://excalidraw.com/#json=vBja0Bv2iL73S_yhzHTPw,aNNm0MlbkZg7t9c36lLcqA)

Reply:
fixed